### DISCUSSION TITLE
Is it possible to assign a channel while creating API?

### DISCUSSION THREAD

**Discussion by chowadfrepo** [saleor/saleor#12610] [General] [answered]
Hi All,

I am trying to create a Product through API but i am not able to assign the product to the channel while creating the product. Is it possible to do it?

**Comment by timuric** [comment, marked as answer]
@chowadfrepo please use https://chrome.google.com/webstore/detail/graphql-network-inspector/ndlbedplllcgconngcnfmkadhokfaaln?hl=en-GB to check how dashboard is interacting with API to enable products in channel
